### PR TITLE
Fix Supermatter Surprise scratch card guaranteed win

### LINKED ISSUE #34745: Getting ripped off by lotto tickets
(WEB REPORT BY: blithering REMOTE: 206.221.180.138:7777)
# Revision
53ed0ac186346646179fe36ac77ac710dddacae0

# Description
Supermatter surprise tickets are supposedly a guaranteed win at the cost of a hand, but they can crop up no winners
# Steps to Reproduce
have hands to spare, scratch off supermatter surprise card
# What you Expected
money no hand
# What Actually Happened
no money no hand

### PR DESCRIPTION
**Pull request by Hinaichigo**
This fixes Supermatter Surprise scratch cards not having a guaranteed win as intended.

Fixes #34745
:cl:
 * bugfix: Supermatter Surprise scratch cards have a guaranteed win as advertised.

### CODE REVIEW

File: code/modules/games/cards/lotto.dm [hunk 1]
```diff
@@ -147,4 +161,4 @@
 	fire_fuel = 1
 
 /obj/item/toy/lotto_ticket/unprinted/attackby(obj/item/weapon/S, mob/user)
-	return 0
```
Comment (Securitymain) [inline code/modules/games/cards/lotto.dm:150]: You return that newline right the fuck now

Comment (Hinaichigo) [reply code/modules/games/cards/lotto.dm:150]: Seems like it gets pruned if you do it in the browser.

**Comment by xeroxemnas** [comment]
based time to do the scratchoffs as a slimeperson and just regrow any lost limbs.

**Comment by falcon2346** [comment]
Why hasn't this been merged? I want to exchange limbs for money dammit

**Comment by Hinaichigo** [comment]
@Kurfursten just a reminder because this one's been skipped on a few rounds of merging.

**Pull request closed without merging**